src/main/java/gift/auth/KakaoAuth.java

```diff
@@ -86,4 +84,42 @@ public String getUserEmail(String accessToken) {
 
         return body.kakaoUserInfo().email();
     }
+
+    public void sendOrderMessage(String accessToken, OrderResponseDto orderResponseDto) {
+        String url = "https://kapi.kakao.com/v2/api/talk/memo/default/send";
+
+        String text = String.format(
+                "주문ID: %d\n옵션ID: %d\n수량: %d\n주문일시: %s\n메시지: %s",
+                orderResponseDto.id(),
+                orderResponseDto.optionId(),
+                orderResponseDto.quantity(),
+                orderResponseDto.orderDateTime(),
+                orderResponseDto.message()
+        );
+
+        Link link = new Link("https://productWeb.com", "https://productMobileWeb.com");
+        KakaoMessageTextTemplateRequestDto textTemplateObject = new KakaoMessageTextTemplateRequestDto("text", text, link, "확인");
+        ObjectMapper objectMapper = new ObjectMapper();
+        String json;
+        try {
+            json = objectMapper.writeValueAsString(textTemplateObject);
+        } catch (JsonProcessingException e) {
+            throw new IllegalArgumentException(e);
```

Comment:
에러 메시지만 생성자 파라미터로 받아도 되지 않을까요? 🤔 
Exception을 통해서 IllegalArgumentException 객체를 만드신 이유가 있으실까요? 

```suggestion
            throw new IllegalArgumentException(e.getMessage());
```

Reply:
그저 예외를 발생시켜야 예외 핸들러가 정의해둔 방법으로 처리한다는 점에 집중하다보니 파라미터에는 신경을 못 쓴거 같습니다.
감사합니다.